lib/message.js

```diff
@@ -121,6 +236,25 @@ var Message = function (client) {
 module.exports = Message;
 
 /**
+ * @class ExtendedMessageResponse
```

Comment:
See how [applications](https://github.com/bandwidthcom/node-bandwidth/blob/master/lib/application.js#L260) handle this to build out the proper jsdoc responses